algos/src/sample_rate_conversion.rst

```diff
@@ -349,94 +343,92 @@ the output is:
 		Arg:         -d -r 32000 -R 48000 -i chirp_test_in.raw -o chirp_test_out.raw -t ../../test/topology/test-playback-ssp2-mclk-0-I2S-src-s32le-s32le-48k-24576k-nocodec.tplg -a src=libsof_src.so -b S32_LE
 		Ld lib path: ../../testbench/build_testbench/sof_ep/install/lib:../../testbench/build_testbench/sof_parser/install/lib
 
-When debugging the testbench library path need to be appended to
-environment variable LD_LIBRARY_PATH and the shown arguments need to
+When debugging the testbench, the library path needs to be appended to
+the environment variable LD_LIBRARY_PATH, and the shown arguments need to
 be set for the debugger such as text mode gdb or graphical ddd. If the
 option to not delete audio data files the test input files can be used
 for debugging as well.
 
-Currently the testbench can be debugged only as host
-(x86) gcc build but debugging possibility with xt-gdb will be restored
-to be able to debug also xtensa optimized version of component in
-testbench.
-   
+Currently the testbench can be debugged only as the host (x86) gcc build but
+debugging possibility with xt-gdb will be restored to be able to debug also
+xtensa optimized version of component in the testbench.
+
 Polyphase decomposition
 ***********************
 
 The SRC component implementation is based on this algorithm level
```

Comment:
Umm, could we say "The SRC component is utilizing algorithm level optimization called polyphase decomposition."?